microsoft-edge/webview2/release-notes/index.md

```diff
@@ -16,18 +16,309 @@ These Release Notes provide information about new features and bug fixes that ar
+
+[NuGet package for WebView2 SDK 1.0.3712-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.3712-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 144.0.3712.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### General changes
+<!-- omit section if empty; usually empty -->
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+The following APIs are in Phase 1: Experimental in Prerelease, and have been added in this Prerelease SDK.
+
+
+<!-- ---------- -->
+###### Allowed port range, transport protocol
+<!-- todo: 2 separate sections? -->
+
+Allowed port range.  Also Transport Protocol.<!-- todo: lead-in -->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- port range -->
+* [CoreWebView2AllowedPortRangeScope Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2allowedportrangescope?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+   * `CoreWebView2AllowedPortRangeScope.Default`
+   * `CoreWebView2AllowedPortRangeScope.WebRtc`
+
+<!-- port range -->
+* `CoreWebView2EnvironmentOptions` Class
+   * [CoreWebView2EnvironmentOptions.GetEffectiveAllowedPortRange Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.geteffectiveallowedportrange?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+   * [CoreWebView2EnvironmentOptions.SetAllowedPortRange Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.setallowedportrange?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+
+<!-- protocol kind -->
+* [CoreWebView2TransportProtocolKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2transportprotocolkind?view=webview2-dotnet-1.0.3712-prerelease&preserve-view=true)
+   * `CoreWebView2TransportProtocolKind.Udp`
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+<!-- port range -->
+* [CoreWebView2AllowedPortRangeScope Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2allowedportrangescope?view=webview2-winrt-1.0.3712-prerelease&preserve-view=true)
+   * `CoreWebView2AllowedPortRangeScope.Default`
+   * `CoreWebView2AllowedPortRangeScope.WebRtc`
+
+<!-- drag? -->
+* [CoreWebView2CompositionController Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2compositioncontroller?view=webview2-winrt-1.0.3712-prerelease&preserve-view=true)<!-- todo: delete?  probably Win32-only, Drag API per https://learn.microsoft.com/microsoft-edge/webview2/release-notes/archive?tabs=win32cpp#customize-the-drag-and-drop-behavior-dragstarting-api -->
+
+<!-- port range -->
+* [CoreWebView2EnvironmentOptions Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.3712-prerelease&preserve-view=true)
+   * [CoreWebView2EnvironmentOptions.GetEffectiveAllowedPortRange Method]<!-- todo -->
```

Comment:
WinRT: add link urls for: 
EnvironmentOptions.GetEffectiveAllowedPortRange
EnvironmentOptions.SetAllowedPortRange